server/src/main/java/org/elasticsearch/index/reindex/ScrollableHitSource.java

```diff
@@ -569,10 +569,19 @@ public String toString() {
         // package-private for testing.
         final long restartFromValue;
 
+        public Checkpoint(StreamInput in) throws IOException {
```

Comment:
I think the changes here turn out unnecessary, see previous comment on the `GET _reindex/id` response contents.